README.md

```diff
@@ -1,2 +1,42 @@
-# silverbackltd-build
-SilverbackLTD Build workflow
+# Silverbackltd Build Workflow
+
+Automatically generate dockerfiles (unless already committed), build docker images and push those images to a configured container registry like GHCR
+
+## Outputs
+
+### `build_images`
+The images that are built and pushed by the workflow
+
+## Example Usage
+
+To push to a ghcr registry:
```

Comment:
make sure to include permissions required

Reply:
ah, good call